src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeService.kt

```diff
@@ -41,10 +42,15 @@ class VilkårperiodeService(
 
     @Transactional
     fun opprettVilkårperiode(behandlingId: UUID, opprettVilkårperiode: OpprettVilkårperiode): Vilkårperiode {
-        feilHvis(behandlingErLåstForVidereRedigering(behandlingId)) {
+        val behandling = behandlingService.hentSaksbehandling(behandlingId)
+        feilHvis(behandling.status.behandlingErLåstForVidereRedigering()) {
```

Comment:
Hvorfor er denne endret?

Reply:
Inlinet `behandlingErLåstForVidereRedigering` då `opprettVilkårperiode` her hadde behov for å sende videre stønadstypen, som finnes i behandlingen